velox/functions/sparksql/tests/ComparisonsTest.cpp

```diff
@@ -73,7 +73,20 @@ class ComparisonsTest : public SparkFunctionBaseTest {
     auto actual = evaluate<SimpleVector<bool>>(
         fmt::format("{}(c0, c1)", functionName), makeRowVector(input));
     facebook::velox::test::assertEqualVectors(expectedResult, actual);
-  };
+  }
+
+  void runAndCompare(
+      const std::string& functionName,
+      const std::vector<VectorPtr>& input,
+      const std::optional<bool>& expectedResult) {
```

Comment:
According to the implementation, there is the hint that input only contains one value. Shall we change input as a single value or support multi-element vector as input? It would be better to make the API and implementation consistent.

Reply:
Updated, thanks.